<?xml version="1.0"?>
<launch>
<param name="use_sim_time" value="false"/>
<arg name="drone_name" default="anafi"/>
<arg name="publish_hz" default="50"/>
  <group ns="$(arg drone_name)">
    <node name="sphinx_update_copter_gazebo_states_from_sphinx_node" pkg="sphinx_with_gazebo" type="sphinx_update_copter_gazebo_states_from_sphinx.py" output="screen" >
        <param name = "publish_hz" value = "$(arg publish_hz)" />
        <param name = "drone_name" value = "$(arg drone_name)" />
    </node> 
  </group>  
</launch>